entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+rm -f /swapfile
+dd if=/dev/zero of=/swapfile bs=1024 count=524288
+mkswap /swapfile
+chmod 600 /swapfile
```

Comment:
This makes me think: this file is not used at all, is it?

You did not tell docker to use it within `Dockerfile` and unless it is a feature of fly.io (I have never used it BTW), this is not used.

Also, If fly.io use this by default, leave the creating of the swapfile here instead of `Dockerfile`. If fly.io doesn't use this by default you can just remove it.

Reply:
@dmitryrck it's used by the production host but yes I didn't end up using this file thanks!